code/game/machinery/mass_driver.dm

```diff
@@ -6,16 +6,26 @@
 	use_power = IDLE_POWER_USE
 	idle_power_usage = 2
 	active_power_usage = 50
+	circuit = /obj/item/circuitboard/machine/mass_driver
 	var/power = 1
 	var/code = 1
 	var/id = 1
-	var/drive_range = 50	//this is mostly irrelevant since current mass drivers throw into space, but you could make a lower-range mass driver for interstation transport or something I guess.
+	var/drive_range = 10	//this is mostly irrelevant since current mass drivers throw into space, but you could make a lower-range mass driver for interstation transport or something I guess.
+	var/power_per_obj = 1000
```

Comment:
I don't quite understand what you mean by this. Do you want me to make the initial values a define? That seems like it would just make the code more cumbersome, Or should drive_range and power_per_object be defines instead of variables? These variables change in RefreshParts() so I don't see that working out either.

Reply:
I think they mean that the initial values should be defines, especially if they are repeated multiple times in the code (in the var definition and in RefreshParts) - this makes it easier to modify all instances of the value at once, makes it clearer what the value is, and makes it harder to put the wrong value in by accident.

Reply:
This is the only place where these literals appear in this context, everything else uses initial(). That's why I said that making these a define would be cumbersome. I suppose I'll do it anyway if it's requested of me though.

Reply:
those are not changed anywhere, they could function as defines.

Reply:
drive_range and power_per_obj both change in RefreshParts().